mockgen/mockgen_test.go

```diff
@@ -335,30 +335,29 @@ func TestGetArgNames(t *testing.T) {
 	}
 }
 
-func Test_lookupPackageName(t *testing.T) {
-	type args struct {
-		importPath string
-	}
+func Test_lookupPackagesName(t *testing.T) {
 	tests := []struct {
-		name            string
```

Comment:
lets keep the same style of test in place with test names and using a runner

Reply:
done